src/modules/SourceManager.rb

```diff
@@ -751,14 +753,9 @@ def InstInitSourceMoveDownloadArea
       download_dir = Ops.greater_than(tmp_space, var_tmp_space) ? "/tmp" : "/var/tmp"
       download_dir = Ops.add(Installation.destdir, download_dir)
       # TODO: check the size of the largest package on CD1
-      successful = Convert.to_integer(
-        SCR.Execute(
-          path(".target.bash"),
-          Builtins.sformat(
-            "test -d '%1' || mkdir -p '%1'",
-            String.Quote(download_dir)
-          )
-        )
+      successful = SCR.Execute(
+        path(".target.bash"),
+        "test -d #{download_dir.shellescape} || /usr/bin/mkdir -p #{download_dir.shellescape}"
```

Comment:
I guess we could drop that `test` as `mkdir -p` does not fail if it already exists.

Reply:
fine for me.